.github/workflows/release-please.yml

```diff
@@ -2,21 +2,23 @@ on:
   push:
     branches:
       - main
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I don't think you want `cancel-in-progress: true` on release PR jobs. If a job is in the process of releasing and someone merges a PR to main the release will get canceled